require.js

```diff
@@ -87,11 +83,17 @@
     if (typeof bundle === 'object') {
       for (var key in bundle) {
         if (has.call(bundle, key)) {
-          modules[key] = bundle[key];
+          require.register(modules[key], bundle[key]);
```

Comment:
Since we're adding a bit more logic, don't duplicate logic, just call the function again with the string/value pair.